packages/aws_common/test/io/aws_file_io_test.dart

```diff
@@ -30,15 +31,19 @@ void main() {
       currentPath = currentPath.replaceAll('/test', '');
     }
     final testFilePath = '$currentPath/test/io/assets/test_file.txt';
+    final testFileWithWrongExtensionPath =
+        '$currentPath/test/io/assets/test_img.txt';
     const testFileContent = 'I ❤️ Amplify, œ 小新';
     final testBytes = utf8.encode(testFileContent);
 
     final testFile = io.File(testFilePath);
+    final testFileWithWrongExtension = io.File(testFileWithWrongExtensionPath);
 
     group('getChunkedStreamReader() API', () {
       test('should return ChunkedStreamReader over io File', () async {
         final awsFile = AWSFilePlatform.fromFile(testFile);
 
+        expect(await awsFile.contentType, lookupMimeType(testFile.path));
```

Comment:
We should assert it's resolving to `text/plain`.

If this is testing `expect(null, null)` we wouldn't know and that wouldn't be very helpful.